massa-ledger-worker/src/ledger_db.rs

```diff
@@ -493,6 +501,20 @@ impl LedgerDB {
         batch.ledger_hash ^= Hash::compute_from(&slot_bytes);
     }
 
+    pub fn get_slot(&self) -> Result<Slot, ModelsError> {
+        let handle = self.db.cf_handle(METADATA_CF).expect(CF_ERROR);
```

Comment:
propogate error instead of panic?

Reply:
I think it's appropriate here because that's totally unrecoverable (the ledger was corrupted), and is done the same way for all other rocks_db calls.

If you think strongly about this maybe we should open an issue on a refactor around this? (E.g. create a new error type to be propagated for all db call fails?)

Reply:
I'm happy with that.